app/models/concerns/group_assignment_and_assignment_scopes.rb

```diff
@@ -18,7 +18,10 @@ module GroupAssignmentAndAssignmentScopes
     scope :start_before, ->(date) { where('period_start < ?', date) }
     scope :start_after, ->(date) { where('period_start > ?', date) }
     scope :start_within, ->(date_range) { where(period_start: date_range) }
-    scope :started_six_months_ago, (-> { where('period_start < ?', 6.months.ago.to_date) })
+    scope :started_six_months_ago, (-> { where('period_start < ?', 6.months.ago) })
+    scope :started_ca_six_weeks_ago, lambda {
+      where('period_start < ? AND period_start > ?', 6.weeks.ago, 8.weeks.ago)
```

Comment:
I saw that in the Story it is written `after "ca" 6 Wochen` but what kind of difference does it make with these two time frames? Can't we just simplify it, and say after 6 Weeks? It will show up anyway after 6 weeks right? So why is there the second date needed?

Reply:
After 6 weeks unlimited will show all (group) assignments that 6 weeks or older. So (group) from the beginning of time until 6 weeks ago.

Reply:
ahso! hmmm so after the 8 weeks, this won't show up in the reminders list huh? 
If we are sure, that they always get feedback within this two weeks frame, then I agree with this. Else, they will loose the overview from the missing feedback, which wasn't given in this two weeks frame..

Reply:
We are not sure. But otherwise it would show assignments for which they don't want/need to send reminders and they would have to delete them in order not to show up again. The time frame and the reminders are not associated with the feedbacks yet, because they don't exist yet. Otherwise we could set as scope at the 6 weeks and modify the reminder criteria to take into account the existence of feedback.